pontoon/base/static/js/main.js

```diff
@@ -220,8 +223,8 @@ var Pontoon = (function (my) {
       });
 
       /* Leading and Trailing spaces */
-      string = markup(string, /^ /gi, 'Leading space');
-      string = markup(string, / $/gi, 'Trailing space');
+      string = markup(string, /^ /gmi, 'Leading space');
+      string = markup(string, / $/gmi, 'Trailing space');
```

Comment:
What's the motivation for adding the `m` modifier here and not in other places within the function?

Should we add a comment?